src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -481,5 +483,26 @@ public interface IHtmlGenerator
             ModelExplorer modelExplorer,
             string expression,
             bool allowMultiple);
+
+        /// <summary>
+        /// Renders HTML markup for the specified partial view.
+        /// </summary>
+        /// <param name="partialViewName">
+        /// The name of the partial view used to create the HTML markup. Must not be <c>null</c>.
```

Comment:
Should probably remove `Must not be null` here or add it to both `ViewContext` and `writer` because we null check those.